kinto/plugins/admin/views.py

```diff
@@ -0,0 +1,26 @@
+import json
+import os
+
+from pyramid.settings import aslist
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+RESPONSE_CACHED = None
```

Comment:
As suggested while pairing, I'd rather use a template here.

```python
INDEX_TEMPLATE = open(os.path.join(HERE, 'build/index.html'), 'r').read()
# Note: I don't remember how you close a file pointer in one call, if it's even possible

def admin_home_view(request):
    # ...
    request.response.write(INDEX_TEMPLATE.replace('<script', globalSettings + '<script'))
    return request.response
```

By the way, it's really odd to mutate `request.response` and return it; is it actually it's supposed to be used?